src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Extensions/OutlineExtensions.cs

```diff
@@ -140,7 +140,7 @@ public static IEnumerable<Breadcrumb> GetBreadcrumbsFromOutLine(this IEnumerable
                 var breadcrumb = new Breadcrumb(item.SeoObjectType)
                 {
                     ItemId = item.Id,
-                    Title = item.SeoInfos?.FirstOrDefault()?.SemanticUrl ?? item.Name,
+                    Title = item.SeoInfos?.FirstOrDefault()?.PageTitle ?? item.Name,
```

Comment:
You must respect the passed  `culture name`  to choose a proper seoInfos object that best matched it, also, you can use this extension https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/VirtoCommerce.ExperienceApiModule.Core/Extensions/IHasLanguageExtensions.cs for that.